code/app/src/main/java/com/example/sigma_blue/fragments/DetailsFragment.java

```diff
@@ -112,6 +123,32 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState)
         tagListAdapter = TagListAdapter.newInstance(currentItem.getTags(), getContext());
         tagListView.setAdapter(tagListAdapter);
 
+        // ITEM IMAGE
+        // trying to get the path of image, and put it on the add item
+        String tempImagePath = globalContext.getCurrentItem().getPhotoPath();
+        // set the image of the item
+        // Create a storage reference from our app
+        if (tempImagePath != null) {
```

Comment:
We should have handling for when the photo path is null. The current interaction is showing a question mark where the photo should be.